master/reference/node/configuration.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Configuring calico/node
+---
+
+The `calico/node` container is primarily configured through environment variables.
+
+
+## Environment Variables
+
+| Environment        | Description                                | Schema | Default |
+|----------------------------------------------------------------|--------|---|--|
+| NODENAME | Node ID. | string | |
```

Comment:
I don't think we should be calling it the ID.

We could say "The name of the node object to use in the Calico API.  Will be created if it does not exist." which is more accurate.